conda-store-server/conda_store_server/orm.py

```diff
@@ -182,6 +197,14 @@ class Build(Base):
     started_on = Column(DateTime, default=None)
     ended_on = Column(DateTime, default=None)
     deleted_on = Column(DateTime, default=None)
+    build_key_version = Column(Integer, default=_get_build_key_version, nullable=False)
+
+    @validates("build_key_version")
+    def validate_build_key_version(self, key, build_key_version):
+        if build_key_version not in [1, 2]:
```

Comment:
This can be a tuple.
```suggestion
        if build_key_version not in (1, 2):
```

But hardcoding these options here (separate from the definitions) is also strange. At this stage we should know which build_key_versions are available via an external name.